src/olympia/activity/models.py

```diff
@@ -673,13 +680,7 @@ def __html__(self):
 
     @property
     def author_name(self):
```

Comment:
This is only still is use at https://github.com/mozilla/addons-server/blob/master/src/olympia/amo/feeds.py, and I'm not exactly sure what it's doing there, and if we could just replace it with `user.name` as I have done in a number of other places.

Reply:
In `feeds.py` it looks like a list of keys, two of which are `author_name` and `item_author_name`. I'm not sure what to replace those with. Can you advise?

Reply:
Seems like the use in feeds.py is unconnected - it predates the reviewer names feature

Reply:
And it's probably unnecessary code anyway - at the time we had rss feeds that contained user submitted content (add-on names, etc) - but now the only rss feed that uses that class is for app versions, that's entirely generated from internal models.

Reply:
Cool, thanks. I wasn't sure if it was related or not. That means we can remove `author_name` from this class.